projects/instant-ui-library/src/lib/button/base-button/base-button.component.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../styles/global.styles";
+@import "../../styles/variables/constants";
+
+$transition-out: ease-in 0.1s;
+$transition-in: ease-out 0.3s;
+$box-shadow-color: none;
+@mixin boxShadow($color) {
+  box-shadow: 0px 2px 8px $color;
+  &:hover,
+  &:focus {
+    box-shadow: 0px 8px 15px $color;
+  }
+}
+@mixin createButton($shadow-color, $text-color) {
+  background-color: $shadow-color;
+  color: $text-color;
+  @include boxShadow($shadow-color);
+  &:disabled {
+    cursor: not-allowed;
+    background-color: $color--disabled3;
+    box-shadow: none;
+    transition: none;
+  }
+}
+.btn {
+  font-size: 12px;
+  border-radius: 6px;
+  padding: $button--padding;
+  font-weight: 700;
+  margin: 5px 5px;
+  cursor: pointer;
+  border: none;
+  border-radius: 8px;
+  transition: $transition-out;
+  &:hover,
+  &:focus {
+    transition: $transition-in;
+  }
+  &:focus {
+    outline: 0.1px solid $color--black;
```

Comment:
**Suggestion:** I think I can see outline related items to be reused. Thoughts on adding as a mixin or variable. Probably create a mixen that adds this whole `&:focus`